src/main/scss/cells/cell.scss

```diff
@@ -100,6 +100,7 @@
     position: relative;
     white-space: nowrap;
     text-overflow: ellipsis;
+    width: 100%;
```

Comment:
Tested in IE?

Reply:
Did not yet have a chance to test on IE.